tests/integration/test_dummy_workflows.py

```
@@ -0,0 +1,37 @@
+from fmcib.run import get_features
+
+
+def test_dummy_features():
+    """
+    Test feature extraction on dummy data.
+
+    Generates a small dummy dataset, extracts features using the model,
+    and verifies that the expected number of features (4096) are produced.
+    """
+    generate_dummy_data("data", size=10)
+    df = pd.read_csv("data/dummy.csv")
```

Ruff

Local variable `df` is assigned to but never used

Remove assignment to unused variable `df`

(F841)